content/en/Integrations/IntegrationBuilder/How to Guides/outlook-notifications.md

```diff
@@ -0,0 +1,98 @@
+---
+title: "Configure Outlook email notifications"
+linkTitle: "Configure Outlook email notifications"
+weight: 40
+---
+
+{{% pageinfo %}}
+How to use Integration Builder to receive email notifications about findings.
+{{% /pageinfo %}}
+
+## Configure Outlook email notifications
+
+In order to configure Outlook to send emails every time a new finding gets created, we need to do the following:
+
+1. Create an Outlook connection.
+2. Find and select a recipe (2 options):
+    - [Cobalt > Outlook] When pentest finding moved to Pending Fix, send email notification
+    - [Cobalt > Outlook] When DAST finding is detected, send email notification
+3. Copy and customize the recipe
+    1. Configure which pentest should trigger notifications when findings get created.
+    2. Configure which email address to send the notification to.
+4. Start the recipe
+
+The following is a step-by-step guide to walk you through the configuration process.
+
+### 1. Create an Outlook connection
+
+The following is a quick guide on how to create a Workato connection for Microsoft Outlook.
+In case you're looking for a more comprehensive and general guide on how to manage connections, check [here](/integrations/integrationbuilder/connect-your-applications/).
+
+Go to the `Integration Builder` > `Projects`. Select or create a project on the left-side of the page. Then click on the "Create" button on the top-right and choose the "Connection" option.
+
+{{% image src="/integrations/integration_builder/findings_outlook_notifications/create_connection_step1.png" alt="Create a connection for your project" %}}
+
+A "New Connection" page will show up. Look for "Outlook" and select the corresponding tile. You can optionally use the search field to quickly find the connection.
+
+{{% image src="/integrations/integration_builder/findings_outlook_notifications/create_connection_step2.png" alt="Pick an Outlook connection" %}}
+
+After selecting the connection type, a form will appear. Provide a name, select a location and click on "Sign in with Windows". If needed, you can configure "Advanced settings" as well.
```

Comment:
> select a location

Is there any particular location we should suggest they put the connection? (typically we put connections in the Cobalt > Connection folder)

Reply:
Yes, I'll add a suggestion to move it into Cobalt > Connection